src/app/games/siam/tests/SiamRules.spec.ts

```diff
@@ -203,34 +195,37 @@ describe('SiamRules:', () => {
             [_, _, U, _, _],
         ];
         const state: SiamState = new SiamState(board, 0);
+        // When trying to perform a fake rotation
+        // TODO TODO FOR REVIEW: what is a fake rotation? The message says "you cannot push and rotate at the same time", but we don't push!
```

Comment:
Le message est bien pourri, mais en fait là il dit "je fais une rotation, oriente moi vers ma direction actuelle", donc une belle rotation ample de 0° (non, ça fait pas froid)
En gros c'est un mouvement sur place, pas explicitement authorisé!

Reply:
ok mais faudrait pas un autre message surtout ? Car là c'est possible de faire ce mouvement dans l'UI, et c'est incompréhensible avec ce message :D

Reply:
Oui, là le message devrais être "you must do a non static move"

Reply:
vu la discussion après, peut-être que c'est OK un message pourri si il peut pas arriver dans l'UI, donc cela sera réglé dans le ticket d'enhancement de Siam ?